graph/db/options.go

```diff
@@ -169,3 +179,5 @@ func WithBatchCommitInterval(interval time.Duration) StoreOptionModifier {
 		o.BatchCommitInterval = interval
 	}
 }
+
+// Todo(abdulkbk) consider adding WithPublicNodeCacheSize.
```

Comment:
Why didn't you add it in this PR, all of the Caches have the possibility to specify the size?